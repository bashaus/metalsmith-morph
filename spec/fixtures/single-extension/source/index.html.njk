<ul>
    <li>{{ templatePath }}</li>
    <li>{{ destinationPath }}</li>
</ul>
