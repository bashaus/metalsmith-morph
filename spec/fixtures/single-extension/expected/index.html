<ul>
    <li>index.html.njk</li>
    <li>index.html</li>
</ul>
